content/hi/docs/reference/glossary/cronjob.md

```diff
@@ -0,0 +1,19 @@
+---
+title: क्रोंजोब (CronJob)
+id: cronjob
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/cron-jobs/
+short_description: >
+  एक दोहराए जाने वाला कार्य (एक काम) जो नियमित समय पर चलता है।
+
+aka:
+tags:
+  - core-object
+  - workload
+---
+
+एक [Job (काम)](/docs/concepts/workloads/controllers/job/) का प्रबंधन करता है जो एक आवधिक अनुसूची पर चलता है।
+
+<!--more-->
+
+एक _crontab_ फ़ाइल में एक पंक्ति के समान, एक क्रॉन्जॉब (CronJob) ऑब्जेक्ट [क्रॉन](https://en.wikipedia.org/wiki/Cron) प्रारूप का उपयोग करके एक शेड्यूल निर्दिष्ट करता है।
```

Comment:
```suggestion
एक _क्रोनटॅब_ फ़ाइल में एक पंक्ति के समान, एक क्रोनजोब ऑब्जेक्ट [क्रॉन](https://en.wikipedia.org/wiki/Cron) प्रारूप का उपयोग करके एक शेड्यूल निर्दिष्ट करता है।
```

Reply:
Amended the cronjob suggestion. I have also kept Crontab as is. Just transliterated it. If we're transliterating Cron job, object, and others, I feel leaving crontab as-is would be a lack of consistency.

Reply:
I'm open to suggestions though. @Garima-Negi @verma-kunal @anubha-v-ardhan @bishal7679 WDYT?

Reply:
@divya-mohan0209 I think it will be better to not to translate `crontab` as we are translating `cronjob`. When any reader will read this two rapidly this might be pronunced the same as `cronjob` for lack of their notice. So if we not translate this it will keep up the uniqueness with `cronjob`. And @Babapool s suggestion also we can consider here.

Reply:
That makes sense, thank you!

Reply:
Translating schedule.
```suggestion
एक _crontab_ फ़ाइल में एक पंक्ति के समान, एक क्रॉन्जॉब (CronJob) ऑब्जेक्ट [क्रॉन](https://en.wikipedia.org/wiki/Cron) प्रारूप का उपयोग करके एक अनुसूची निर्दिष्ट करता है।
```